modules/system/classes/SystemController.php

```diff
@@ -35,9 +36,34 @@ public function combine($name)
             $combiner = CombineAssets::instance();
 
             return $combiner->getContents($cacheId);
-        }
-        catch (Exception $ex) {
+        } catch (Exception $ex) {
             return Response::make('/* '.e($ex->getMessage()).' */', 500);
         }
     }
+
+    /**
+     * Resizes an image using the provided configuration
+     * and returns a redirect to the resized image
+     *
+     * @param string $identifier The identifier used to retrieve the image configuration
+     * @param string $encodedUrl The double-encoded URL of the resized image, see https://github.com/octobercms/october/issues/3592#issuecomment-671017380
+     * @return RedirectResponse
+     */
+    public function resizer(string $identifier, string $encodedUrl)
+    {
+        $resizedUrl = ImageResizer::getValidResizedUrl($identifier, $encodedUrl);
+        if (empty($resizedUrl)) {
+            return response('Invalid identifier or redirect URL', 400);
+        }
```

Comment:
@bennothommo could we get a unit test for both of these code paths (correctly signed URL being validated vs incorrectly signed URL being rejected)?